### PR TITLE
Added forcebalance

### PR DESCRIPTION
**Pull request by leeping**

### CODE REVIEW

File: forcebalance/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package:
+  name: forcebalance-dev
```
Comment (Lnaden) [inline forcebalance/meta.yaml]: Do you actually want to call this `forcebalance-dev`? That is how people will locate the package, they will have to call `conda install forcebalance-dev` to access it. 

If you want to upload this to the `dev` label, we have another way to do it.

File: forcebalance/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  version: "1.5.1"
+
+source:
+  url: https://github.com/leeping/forcebalance/archive/v1.5.1.tar.gz
+  fn: v1.5.1.tar.gz
+
+build:
+  number: 0
+  skip: True # [py2k]
```
Comment (Lnaden) [inline forcebalance/meta.yaml]: I know FB recently became Py3 compatible, but do you truly want to skip the Python 2.7 builds?

Comment (leeping) [reply forcebalance/meta.yaml]: Oops - I didn't know that's what it meant!

File: forcebalance/meta.yaml (hunk 1 above)
Comment (leeping) [reply forcebalance/meta.yaml]: I'll remove the -dev label now.

File: forcebalance/meta.yaml (hunk 2 above)
Comment (leeping) [reply forcebalance/meta.yaml]: Should this be changed to `skip: False`?

File: forcebalance/meta.yaml (hunk 1 above)
Comment (Lnaden) [reply forcebalance/meta.yaml]: We can have it upload to the `dev` label pretty easily actually, if you don't want it on the `main` channel

File: forcebalance/meta.yaml (hunk 2 above)
Comment (Lnaden) [reply forcebalance/meta.yaml]: You can just remove this line if you dont want any builds skipped. Its optional

**Pull request merged**